types/atlassian-connect-js/package.json

```diff
@@ -12,10 +12,6 @@
         {
             "name": "Josh Parnham",
             "githubUsername": "josh-"
-        },
-        {
-            "name": "Tobias Theobald",
-            "githubUsername": "Tobi042"
```

Comment:
@josh- Sorry, I didn't realize my username change would trigger this 🤦 Yeah, if it's fine with you, I'd like to stay on the list. I don't often have time look at this stuff, but I do feel I can still contribute something and having more than one person on this list reduces the bus factor.

Reply:
no worries at all! would definitely appreciate having another person on the maintainers list so that sounds great 🙂

@jakebailey would you be able to update this username? otherwise happy to open a PR.

Reply:
No worries, this is normal when accounts change, some people genuinely don't want to be readded or forgot they were even owners.